crates/provider/src/provider/trait.rs

```diff
@@ -34,19 +34,11 @@ use super::EthCallMany;
 /// See [`PollerBuilder`] for more details.
 pub type FilterPollerBuilder<R> = PollerBuilder<(U256,), Vec<R>>;
 
-// todo: adjust docs
-// todo: reorder
-/// Provider is parameterized with a network and a transport. The default
-/// transport is type-erased, but you can do `Provider<Http, N>`.
+/// Provider is parameterized with a network. The default
+/// transport is type-erased, but you can do `Provider<N>`.
```

Comment:
we can change this to "Ethereum JSON-RPC interface." and remove the second sentence